packages/@aws-cdk-testing/framework-integ/test/aws-eks/test/sdk-call-integ-test-docker-app/app/package-lock.json

```diff
@@ -6,7 +6,7 @@
     "": {
       "name": "eks-service-account-sdk-call-integ-test",
       "dependencies": {
-        "@aws-sdk/client-s3": "^3.408.0"
```

Comment:
I updated package-lock.json since it was not updated.

Reply:
I saw that you opened a separate [PR](https://github.com/aws/aws-cdk/pull/27686) for this, which is great.
If the build does not fail can you remove the changes to this and the corresponding Dockerfile so we can keep them separate?

Reply:
Actually, the build fails with the original implementation. I'm not sure if this issue will be merged, so I created another PR. What should we do?

Reply:
Ok, just align the node version to `node:18-alpine3.18` here as well so if this gets merged the other one can just be closed.

Reply:
Yes, I changed it to `node:18-alpine3.18`.